File: Programming courses/C:C++/Klausur/WS1920/a5/Person.cpp
#include "Person.h"
#include "Ort.h"
#include <string>

Person::Person(Ort * geb, Ort * wohn, int alter, std::string name):geburtsort{geb},wohnort{wohn},alter{alter},name{name}{}

Person::~Person(){
    delete[] wohnort;
    delete[] geburtsort;
}

void Person::setAlter(int x){
    this->alter = x;
}

const std::string Person::getName(){
    return this->name;
}

File: Programming courses/Java/02/ab/ab9/src/Person.java
class Person {
    private String name;
    private int alter;
    public Person(String name, int alter) {
        this.name= name;
        this.alter= alter;
    }
    public String getName() { 
        return name;
    }
    public int getAlter() { 
        return alter;
    }
    @Override public String toString() {
        return name + " (" + alter +")";
    }
}